### PR TITLE
Added new How To (simple io with NumPy)

### PR DESCRIPTION
**Pull request by melissawm**
Hi all,

This is the draft for the How to on storing and reading arrays from disk with NumPy. A couple of comments:
1) This does not cover all topics raised in [this NumPy issue](https://github.com/numpy/numpy/issues/15760). That was too much for one How To. The current plan is to do another How To specifically mentioning cloud storage and more complicated issues, and an "Explanation" or "Discussions" page on the NumPy storage model and advanced performance comments.
2) There is definitely a bit more discussion than what would be recommended by the [How To guide](https://documentation.divio.com/) we're using. I'm aware of that, and I'm happy to take feedback on this, but I just couldn't see how to explain all this without some minor explanations.

Suggestions and feedback are much appreciated!

### CODE REVIEW

**Comment by eric-wieser** [comment]
@rossbar, was the intent to use the mystnb format here for a better plaintext view when reviewing?

File: content/howto-simple_io.ipynb [hunk 1]
```diff
@@ -0,0 +1,834 @@
...
+   "source": [
+    "<a id=\"example\"></a>\n",
+    "## Example: Reading a netCDF file data and saving it as a NumPy array"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this example, we'll take a file from [gridMET](http://www.climatologylab.org/gridmet.html) containing meteorological data from the continental United States. \n",
```
Comment (eric-wieser) [inline content/howto-simple_io.ipynb:41]:
```suggestion
    "For this example, we'll take a file from [gridMET](https://www.climatologylab.org/gridmet.html) containing meteorological data from the continental United States. \n",
```

Comment (eric-wieser) [reply content/howto-simple_io.ipynb:41]: Nevermind, not valid

File: content/howto-simple_io.ipynb [hunk 2]
```diff
@@ -0,0 +1,834 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that `air_temperature` is a [netCDF variable](https://www.unidata.ucar.edu/software/netcdf/docs/group__variables.html). We can read it as a NumPy array using the following syntax:\n",
+    "```python\n",
+    "temperatures = air_temperature[:]\n",
+    "```\n",
+    "\n",
+    "To simplify our example, we'll read only the temperature data between July 1st and July 31st, 2019. These are indexed by 181 and 211, respectively, in the first axis of our `air_temperature` array (remember that when slicing the stop value - 212 in the expression below - is not included in the final result.)"
```
Comment (eric-wieser) [inline content/howto-simple_io.ipynb:166]: It might be good to show where 181 and 212 came from, or a way of finding them programmatically.

**Comment by rossbar** [comment]
> was the intent to use the mystnb format here for a better plaintext view when reviewing?

The primary goal of using something like `myst-nb` for the `numpy-tutorials` repo is to allow users to submit content in formats other than rST. The text-based notebook format is a nice feature, but I don't think the `numpy-tutorials` repo will have a policy recommending one format over any other. Hopefully the text-based format catches on with Jupyter users and we start seeing submissions with that format.

**Comment by melissawm** [comment]
Thanks for the comments. Since some of the discussion has moved to #15 , it might be worthwile continuing there and closing (or at least pausing) this issue. If there is still value in this document after that, I'll retake it.

**Pull request closed without merging**